pact/Pact/Core/Gas/TableGasModel.hs

```diff
@@ -215,6 +248,43 @@ runTableModel = \case
      quadraticGasFactor = 50_000
      linearGasFactor = 38_000
   GModuleMemory bytes -> moduleMemoryCost bytes
+  GStrOp op -> case op of
+    StrOpLength len ->
+      let charsPerMg = 100
+      in MilliGas $ fromIntegral $ len `quot` charsPerMg + 1
+    StrOpConvToInt len ->
+      let mgPerChar = 20
+      in MilliGas $ fromIntegral $ len * mgPerChar + 1
+    StrOpParse len ->
```

Comment:
This one is definitely interesting. 

Doesn't this highly depend on the parser used?